design/src/high-level/creative-automation-case-studies.md

```diff
@@ -0,0 +1,282 @@
+# Creative Automation: Case Studies
+
+## Emergence
+
+**Conclusion:** Creativity will be low until more systems are in place, but potential is excellent. Clear and satisfying aesthetics will be key to creating a compelling contraption. Disruptions should be tunable to ensure every player can reach an optional pressure level.  
+
+### Interesting Goals
+
+Standard factory building goals are present,
+but an emphasis on sustainability and resilience add even more goals to consider.
+
+Stronger spatial and temporal variability increase the importance of considering tradeoffs and tailoring solutions to conditions.
+
+### Distinct Paths
+
+Initial prototypes will be quite weak, simply due to scale.
+
+Adding more systems and organisms will enhance this substantially.
+
+### Low Pressure
+
+Aesthetics are well-suited to chill alien cottage-core vibes.
+Disruptions are planned as an important gameplay mechanic: need excellent tuning levers to ensure that space for creativity isn't removed.
+Cascading failures shouldn't be too severe by default.
+
+### Compelling Aesthetics
+
+Smashing success.
+Unique fantasy that lends itself well to a bustling, automated hub.
+
+Aesthetic differentation will require more content than MVP.
+
+Potential for very compelling contraptions.
+Audio and timing and visual clarity will be tricky but essential to creating good contraptions.
+
+## [Factorio](https://store.steampowered.com/app/427520/Factorio/)
+
+**Conclusion:** top-tier creative automation, especially in modded games.
+Lots of options that differ in interesting ways.
```

Comment:
Mods don't change aesthetics at all, is my guess? (sorry, sorry, I *still* haven't bought Factorio!)

Reply:
I haven't really played with mods myself yet, but I think they can add additional graphics (e.g. for new machines).
I'm not entirely sure if they can change the graphics of existing entities.

Reply:
They can, even for existing buildings :)